regress/expected/expr.out

```diff
@@ -5117,6 +5117,59 @@ ERROR:  schema "contains" does not exist
 LINE 2:     RETURN contains.age_sqrt(25)
                                     ^
 --
+-- user defined function expressions - testing for age_cbrt function
+--
+SELECT * from cypher('expr', $$
+    RETURN ag_catalog.age_cbrt(25)
+$$) as (result agtype);
+      result      
+------------------
+ 2.92401773821287
+(1 row)
+
+SELECT * from cypher('expr', $$
+    RETURN pg_catalog.cbrt(25::pg_float8)
+$$) as (result agtype);
+      result      
+------------------
+ 2.92401773821287
+(1 row)
+
+-- should return null
+SELECT * from cypher('expr', $$
+    RETURN pg_catalog.cbrt(null::pg_float8)
+$$) as (result agtype);
+ result 
+--------
+ 
+(1 row)
+
```

Comment:
add a null call without the typecast

Reply:
For the pg_catalog.cbrt or the ag_catalog.age_cbrt

Reply:
ag_catalog In general. Thats what I care about